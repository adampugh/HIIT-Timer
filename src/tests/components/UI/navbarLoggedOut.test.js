import React from "react";
import { shallow } from "enzyme"; 

import Navbar from "../../../components/UI/navbarLoggedOut";

test("should render navbarLoggedOut correctly", () => {
    const wrapper = shallow(<Navbar />);
    expect(wrapper).toMatchSnapshot();
});

test("should call handleOpenModal in navbarLoggedOut", () => {
    const handleOpenModalSpy = jest.fn();
    const wrapper = shallow(<Navbar handleOpenModal={handleOpenModalSpy} />)
    wrapper.find('h3').simulate("click");
    expect(handleOpenModalSpy).toHaveBeenCalled();
});