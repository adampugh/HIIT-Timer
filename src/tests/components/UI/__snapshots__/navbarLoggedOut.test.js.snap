// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbarLoggedOut correctly 1`] = `
<div>
  <loginModal />
  <nav
    className="navbar navbar--loggedOut"
  >
    <div
      className="container"
    >
      <div
        className="navbar__items"
      >
        <h2>
          HIIT Timer
        </h2>
        <div
          className="navbar__links"
        >
          <h3>
            Sign Up / Log In
          </h3>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
